<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistWindow</class>
 <widget class="QMainWindow" name="RegistWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>710</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1018</width>
    <height>710</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1018</width>
    <height>710</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{border-image:url(:/images/background.jpg)}
QMessageBox{border-image:none}
QLabel{border-image:none}
QPushButton{border-image:none}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>350</width>
      <height>150</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>150</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>150</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
color:rgb(255,255,255);
font:100 20pt &quot;微软雅黑&quot;;
bottom:10px

</string>
    </property>
    <property name="text">
     <string>账号注册！</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>471</width>
      <height>491</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLineEdit {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
	font: 10pt &quot;微软雅黑&quot;
}
QComboBox {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
	font: 13pt &quot;微软雅黑&quot;;
}
QFrame{border-radius: 5px};
background-color:rgb(255, 255, 255);
border-image:none;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>390</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 170, 255);
font: bold 10pt &quot;微软雅黑&quot;;
color:rgb(255,255,255);
border-radius: 5px;
}QPushButton::hover{
background-color:rgb(0, 150, 225);
}
QPushButton::pressed{
background-color:rgb(0, 150, 225);
}</string>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>390</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(0, 170, 255);
font: bold 10pt &quot;微软雅黑&quot;;
color:rgb(255,255,255);
border-radius: 5px;
}QPushButton::hover{
background-color:rgb(0, 150, 225);
}
QPushButton::pressed{
background-color:rgb(0, 150, 225);
}</string>
     </property>
     <property name="text">
      <string>重置</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_33">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
font:100 13pt &quot;微软雅黑&quot;;
bottom:10px

</string>
     </property>
     <property name="text">
      <string>姓名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_40">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
font:100 13pt &quot;微软雅黑&quot;;
bottom:10px

</string>
     </property>
     <property name="text">
      <string>电话：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
font:100 13pt &quot;微软雅黑&quot;;
bottom:10px

</string>
     </property>
     <property name="text">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
font:100 13pt &quot;微软雅黑&quot;;
bottom:10px

</string>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_36">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_37">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_38">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>320</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:none;
font:100 13pt &quot;微软雅黑&quot;;
bottom:10px

</string>
     </property>
     <property name="text">
      <string>确认密码：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_39">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
